### ISSUE TITLE
BD crashing whenever i go to open it. I reinstall it over and over again but everytime it says to disable plugins before starting BD. Where is the plugins folder Location?

### ISSUE THREAD

**Issue by xFrxdx** [BetterDiscord/BetterDiscord#1034]

**Comment by fowlis** [comment]
Duplicate of #995 
Solution: [here](https://github.com/BetterDiscord/BetterDiscord/issues/995#issuecomment-901337922)
If that solution doesn't work, join the community Discord for more help: [invite](https://discord.gg/bd-app)

**Comment by xFrxdx** [comment]
that was the fix thank you
